corehq/apps/linked_domain/static/linked_domain/js/domain_links.js

```diff
@@ -75,6 +75,29 @@ hqDefine("linked_domain/js/domain_links", [
             return DomainLink(link);
         }));
 
+        self.linkable_ucr = ko.observableArray(_.map(data.linkable_ucr, function (report) {
+            return RemoteLinkableReport(report, self.master_link);
+        }));
+        self.createRemoteReportLink = function (reportId) {
+            _private.RMI("create_remote_report_link", {
+                "master_domain": self.master_link.master_domain,
+                "linked_domain": self.master_link.linked_domain,
```

Comment:
there will be an overhaul in the future, but if it isn't too much of a pain to change `upstream_domain` and `downstream_domain` that would be ideal.